Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["service", "program-keccak-inclusion", "common", "blob-tool", "runner-keccak-inclusion", "eqs-client"]
+members = ["service", "program-keccak-inclusion", "common", "blob-tool", "runner-keccak-inclusion", "eqs-client", "zksync-interface", "sdk"]
```

Comment:
Sry this is build on #47 , please ignore new crates around this in review (if not removed from commits when you do)